cmd/operator/main.go

```diff
@@ -26,11 +26,12 @@ func main() {
 	log.Infof("starting Dapr Operator -- version %s -- commit %s", version.Version(), version.Commit())
 
 	ctx := signals.Context()
-	kubeClient, daprClient, err := k8s.Clients()
+	clients, err := k8s.NewClients(nil, nil)
```

Comment:
This is not usually common go use a New method and passing nils into it..

Usually, New methods take no arguments and take care of initialization internally.
Its ok to pass arguments when needed, but when both arguments are nil, the question is asked, why have these there in the first place.

A refactor would be to have `k8s.NewClients()`

Reply:
I have made the type Clients to Client. Since it is abstraction and internal implementation takes care. So I think it is better to keep the type Client .  I have changed it and pushed the change.

Or we need to totally give a different name other than client/clients to something like KubeInteractor or just Interactor. Please let me know.